### DISCUSSION TITLE
APIs

### DISCUSSION THREAD

**Discussion by egecans** [bounswe/bounswe2023group2#157] [Ideas]
We need to choose useful APIs that will work on this project. Therefore we have to learn their aims, advantages and disadvantages very clearly. Below this topic; you can discuss which API we can use for which functions, additionally you can share beneficial links to learning APIs.

**Comment by bgmrsln** [comment]
If you never used APIs and want to have a better idea about it you can watch [APIs for Beginners](https://www.youtube.com/watch?v=WXsD0ZgxjRw). It is a 3 h  video, but it seems to be good.
